src/handlers/dashboard.rs

```diff
@@ -9,18 +11,14 @@ use axum::{
     Extension,
 };
 
-use crate::common::repository::DbReadAll;
-
+/// # Panics
```

Comment:
When does it panic? Isn't it because the DbReadAll isn't imported?

Reply:
Probably related to the unhandled unwraps

Reply:
The comment was auto added by clippy